index.d.ts

```diff
@@ -17,9 +17,7 @@ declare module 'data-store' {
     path?: string
   }
 
-  interface DataObject {
-    [key: string]: any
-  }
+  type DataObject = Record<string, any>;
```

Comment:
Where does `Record` come from? Is this built-in TypeScript? If so, which version is required?

Reply:
`Record` is a builtin type, available since TypeScript 2.1.
https://unpkg.com/browse/typescript@2.1.6/lib/lib.es5.d.ts#L1402